packages/components/src/components/remaining-time/remaining-time.tsx

```diff
@@ -1,9 +1,16 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { formatDuration, getDiffDuration } from '@deriv/shared';
+import moment from 'moment';
 
-const RemainingTime = ({ end_time = null, format, getCardLabels, start_time }) => {
-    if (!+end_time || start_time.unix() > +end_time) {
+type TRemainingTimeProps = {
+    end_time?: number | null;
+    start_time: moment.Moment;
+    format: string;
+    getCardLabels: () => { [key: string]: string }; // TODO Use the one from common after contract-card migration
```

Comment:
I think using Record utility type would be the cleaner choice.
Record<string, string>